src/components/Login.js

```diff
@@ -46,7 +46,7 @@ class Login extends Component {
     if (result === undefined) {
       Actions.login();
     } else {
-      Actions.main();
+      Actions.generalInfo();
```

Comment:
I think that there is no difference about performance.
Does anybody know about that?